### PR TITLE
copyright and authors

### PR DESCRIPTION
**Pull request by akiezun**
@eitanbanks please review and merge if OK

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+# For example, Broad Institute employees are not listed here, 
+# because Broad Institute holds the copyright.
+
+# Names should be added to this file as
+# Name or Organization <email address>
+# The email address is not required for organizations.
+
+# Please keep the list sorted.
+Broad Institute Inc.
+Nicholas Newell <nenewell@comcast.net>
```
Comment (eitanbanks) [inline AUTHORS]: Please add "Intel Corp."
(They will want this)

Comment (lbergelson) [inline AUTHORS]: Shouldn't google go in here now?  They have 1 commit so far.

File: AUTHORS [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+# The AUTHORS file lists the copyright holders.
+# For example, Broad Institute employees are not listed here, 
+# because Broad Institute holds the copyright.
+
+# Names should be added to this file as
+# Name or Organization <email address>
+# The email address is not required for organizations.
+
+# Please keep the list sorted.
+Broad Institute Inc.
```
Comment (eitanbanks) [inline AUTHORS]: Sorry but there needs to be a comma after the word "Institute".  I know we are talking minutia but I don't want to get this one wrong.  :)

**Comment by eitanbanks** [comment]
Review complete.

File: AUTHORS (hunk 1 above)
Comment (akiezun) [inline AUTHORS]: added Intel

File: AUTHORS (hunk 2 above)
Comment (akiezun) [inline AUTHORS]: added comma

**Comment by akiezun** [comment]
back to @eitanbanks 
(sorry i squashed too early.)

**Pull request merged**